src/components/screens/PointOfInterest.js

```diff
@@ -210,9 +210,7 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
                 [category.iconName || MAP.DEFAULT_PIN]: 1,
                 iconName: category.iconName || MAP.DEFAULT_PIN,
                 id,
-                position: { latitude, longitude },
-                serviceName: title,
-                title
```

Comment:
we need them only for callouts and we have no callouts in poi detail screen, that is why we can remove them here?

Reply:
I don't quite understand the question. I already removed them because we don't need CalloutText, so why are you asking if we can remove them again?

Reply:
the question was, why we can remove them and i tried to answer it myself. do you agree with "because it is just for callouts and we have no callouts at POI details"?

Reply:
Yes, absolutely. https://github.com/smart-village-solutions/smart-village-app-app/pull/1328#discussion_r2086113956 This is exactly what I was trying to say in the comment here.